docs/workflows/phylogenetic_construction/snippy_variants.md

```diff
@@ -68,6 +68,7 @@ The `Snippy_Variants` workflow aligns single-end or paired-end reads (in FASTQ f
 
 | **Variable** | **Type** | **Description** |
 |---|---|---|
+| snippy_combined_qc_metrics | File | Combined QC metrics file containing concatenated QC metrics from all samples. The file is a tab-separated values (TSV) file with the following columns:<br>- samplename<br>- reads_aligned_to_reference<br>- total_reads<br>- percent_reads_aligned<br>- variants_total<br>- percent_ref_coverage<br>- #rname<br>- startpos<br>- endpos<br>- numreads<br>- covbases<br>- coverage<br>- meandepth<br>- meanbaseq<br>- meanmapq<br><br>The last set of columns (`#rname` to `meanmapq`) may repeat for each chromosome or contig in the reference genome. |
```

Comment:
This isn't the right output column for this workflow -- looks like it's the unconcatenated version `snippy_variants_qc_metrics`. Could you please update this?

Reply:
Looking at this further, could you consider moving the description of the QC metric values to the Workflow Tasks section for Snippy_Streamline, Snippy_Streamline_FASTA, Snippy_Tree, and Snippy_Variants? The amount of information in this cell in the table makes it appear bloated. You could then reference people in this table to look at the appropriate section in the docs

For example, you could add a few lines called "QC Metrics" at line 171 of snippy_streamline.md where you describe the contents of the snippy_combined_qc_metrics file. You'd copy that to the appropriate places in Snippy_Streamline_FASTA and Snippy_Tree. In Snippy_Variants, though, you'd add that section around line 60 in the workflow tasks subheading.

Happy to chat about this in more detail if this is unclear

Reply:
See if you agree with my documentation changes